### PR TITLE
Update data.fr-FR.json

### PR DESCRIPTION
**Pull request by MaelREBOUX**
first corrections related to #86

### CODE REVIEW

File: assets/translations/data.fr-FR.json [hunk 1]
```diff
@@ -72,18 +72,18 @@
...
+                "printError": "Impossible d'imprimer les données du formulaire de demande.",
+                "availableReqError": "Impossible de récupérer les demandes disponibles."
             },
             "cadastre_tools" : "Outils Cadastre",
-            "create_line": "Sélectionne / Active / Désélectionne une ou plusieurs parcelles par une ligne brisée",
-            "create_point": "Sélectionne / Active / Désélectionne une parcelle par un simple clic",
-            "create_polygon": "Sélectionne / Active / Désélectionne une ou plusieurs parcelles à l’intérieur d’un polygone",
+            "create_point": "Sélectionner une parcelle par un simple clic",
+            "create_line": "Sélectionner des parcelles par une polyligne",
+            "create_polygon": "Sélectionnee des parcelles par un polygone",
```
Comment (landryb) [inline assets/translations/data.fr-FR.json:83]: +1000 for the intent ! but typo on the last one..

Comment (offtherailz) [reply assets/translations/data.fr-FR.json:83]: If correct, I can merge it

Comment (landryb) [reply assets/translations/data.fr-FR.json:83]: the last one should be `Sélectionner` not `Sélectionnee`

Comment (offtherailz) [reply assets/translations/data.fr-FR.json:83]:
```suggestion
            "create_polygon": "Sélectionner des parcelles par un polygone",
```

Comment (offtherailz) [reply assets/translations/data.fr-FR.json:83]: @landryb I'm going to merge this PR and fix this string in a separate update, to speed up the process

**Pull request merged**